nav2_simple_commander/nav2_simple_commander/costmap_2d.py

```diff
@@ -0,0 +1,73 @@
+#! /usr/bin/env python3
+# Copyright 2021 Samsung Research America
+# Copyright 2022 Stevedan Ogochukwu Omodolor
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+
+
+class PyCostmap2D:
+    """
+    PyCostmap2D.
+
+    Costmap API for OccupancyGrid messages coming into the Simple Commander API
+    """
+
+    def __init__(self, occupancy_map):
+        self.size_x_ = occupancy_map.info.width
```

Comment:
I don't think in python we typically do trailing underscores - isn't `self.` already an indicator of a member variable?

Reply:
Its an old habit I picked up from cpp😅, I can change it to the other convention if needed. In theory as you said, self do indeed indicate that its part of the class.